docs/core/encoding.md

```diff
@@ -197,7 +197,8 @@ The above `Profile` example is a fictive example used for educational purposes.
 - the `sdk.Msg` interface for encoding different `Msg`s in a transaction,
 - the `AccountI` interface for encodinig different types of accounts (similar to the above example) in the x/auth query responses,
 - the `Evidencei` interface for encoding different types of evidences in the x/evidence module,
-- the `AuthorizationI` interface for encoding different types of x/authz authorizations.
+- the `AuthorizationI` interface for encoding different types of x/authz authorizations,
+- the `Validator` [object](https://github.com/cosmos/cosmos-sdk/blob/v0.42.5/x/staking/types/staking.pb.go#L306-L337) which is fully encoded as `Any`. 
```

Comment:
One small fix regarding the description here. We already know the validator struct is full encoded as `Any` given the context. We should probably describe the validator struct.
```suggestion
- the [`Validator`](https://github.com/cosmos/cosmos-sdk/blob/v0.42.5/x/staking/types/staking.pb.go#L306-L337) struct that contains information about a validator. 
```